packages/database/src/schema.prisma

```diff
@@ -606,6 +608,31 @@ model StatementDocument {
   Document        Document        @relation(fields: [documentId], references: [id], onDelete: NoAction, onUpdate: NoAction)
 }
 
+// Stores appeal final comments from either an LPA, appellant or rule 6 group
+model AppealFinalComment {
```

Comment:
Can we rename this to just FinalComment
Nice to have the Join table's name be a straight combination of the two tables it's linking between

Reply:
I think they will go in as separate tables?

Reply:
Ah ok, I misunderstood this

> Nice to have the Join table's name be a straight combination of the two tables it's linking between

to mean having the join tables as one, as the table names were already named as comment/statementDocument just needed to rename the comment join table to match

Reply:
sorry yes think I was talking about the fact it was:
AppealFinalComment 
FinalCommentDocument

To change to
FinalComment
FinalCommentDocument

Sorry for misunderstanding and extra work

Reply:
Yea no worries, reading it back now it makes sense. Will fix it